### DISCUSSION TITLE
Npx create-next-app@latest command not working in MacBook Air m2

### DISCUSSION THREAD

**Discussion by Aaravkapoor1** [vercel/next.js#68385] [Help]
### Summary

<img width="1470" alt="Screenshot 2024-08-01 at 1 08 03 PM" src="https://github.com/user-attachments/assets/f91e331c-b958-4c1b-813e-cbd6f8dcc1c0">

**Comment by icyJoseph** [comment]
Something has gotten corrupted, I think, in your globally installed packages. Try to uninstall and reinstall:

```console
npm uninstall -g create-next-app
```
